presto-native-execution/presto_cpp/main/types/tests/PlanConverterTest.cpp

```diff
@@ -75,15 +76,11 @@ std::shared_ptr<const core::PlanNode> assertToBatchVeloxQueryPlan(
 
```

Comment:
Shall we have two convert utility and one for write and the other for read? Thanks!

Reply:
l78. assertToBatchVeloxQueryPlan used to create a batch plan with writer (mapper side), and this PR changes to a batch plan with reader (reducer side). Do we want to keep testing both? Thanks!